docs/deploy-rbd.md

```diff
@@ -134,7 +134,7 @@ for more information.
 **Deploy Ceph configuration ConfigMap for CSI pods:**
 
 ```bash
-kubectl create -f ../example/ceph-config.yaml
+kubectl create -f ../../../examples/ceph-conf.yaml
```

Comment:
Loos good to me, thanks!

I'm just wondering why this file is not included in the `deploy/...` directory. Would it make sense to move/copy it there?

Reply:
I do not have any idea why it has been kept outside the `deploy/` directory. If there is no solid reason for keeping it in the `examples/` directory and everyone agrees on this change, will move this to `/deploy/` dir, which needs updates in few other files as well. Or I can copy it to `deploy/` dir, so that we can avoid some diffs.

Reply:
it was kept in example folder when we started initially, common files like [service-monitor.yaml](https://github.com/ceph/ceph-csi/blob/devel/examples/service-monitor.yaml), [csi-config-map-sample.yaml](https://github.com/ceph/ceph-csi/blob/devel/examples/csi-config-map-sample.yaml) and [ceph-conf.yaml](https://github.com/ceph/ceph-csi/blob/devel/examples/ceph-conf.yaml) can be moved to deploy folder and the documentation and e2e test cases need to be adjusted for the same.

Reply:
Thanks @Madhu-1 for the background.
Just to confirm, you are asking to move all those 3 common files under `deploy/` or just the ceph-conf.yaml file?

Reply:
yes as its applicable to both cephfs/nfs and rbd you can move these 3 files to deploy folder

Reply:
Done.